tyrian/js/src/main/scala/tyrian/websocket/WebSocket.scala

```diff
@@ -105,28 +106,36 @@ object WebSocket:
     Async[F].delay {
       val socket    = new dom.WebSocket(address)
       val keepAlive = new KeepAlive(socket, settings)
-      val q         = Queue.empty[WebSocketEvent]
+      val q         = Queue.unbounded[F, WebSocketEvent]
+
+      val enqueue: WebSocketEvent => Unit = wse =>
+        Dispatcher.sequential[F].use { disp =>
+          Async[F].delay {
+            val action = Async[F].flatMap(q)(_.offer(wse))
+            disp.unsafeRunAndForget(action)
+          }
+        }
```

Comment:
I think I know why your PR didn't work :) there's a couple issues with this code:
1. it's opening/closing the dispatcher (the `use`) before the submitted action has a chance to run
2. you are never actually running this effect!

The motivation behind this PR is excellent btw, we just need to adjust the approach a bit. Basically, you want to create a `Dispatcher` once, and re-use it for all events. Here's an example.

https://github.com/http4s/http4s-dom/blob/8734cdc909063a157bca1f95b625032bb5e63cab/dom/src/main/scala/org/http4s/dom/WebSocketClient.scala